__init__.py

```diff
@@ -280,6 +290,14 @@ def draw(self, context):
 	Fast64_GlobalToolsPanel,
 )
 
+def derive_legacy_defaults():
+	'''Set scene properties after a scene loads, used for migrating old properties'''
+	SM64_Properties.derive_defaults()
```

Comment:
I don't think "derive defaults" is that great of a name, this is about "upgrading"/migrating properties. Can't really think of a name I like atm

Reply:
changed it to `upgrade_changed_props`